packages/ui-popover/src/Popover/README.md

```diff
@@ -170,6 +226,148 @@ class Example extends React.Component {
 render(<Example />)
 ```
 
+#### Popover playground
```

Comment:
This looks pretty cool now too! I suggest putting the color picker input here too (the `primary` and `primary-inverse` radio you put in the Uncontrolled Popover example too)! :)

Reply:
Great idea! I will do it soon